src/pages/Models/Details/ModelDetails.js

```diff
@@ -269,51 +200,44 @@ const ModelDetails = () => {
   const handleAppRowClick = (e, app) => {
     const currentApp = cloneDeep(app);
     currentApp.name = e.currentTarget.dataset.app;
-    setSlidePanelData({ currentApp });
     setFilterByApp(currentApp.name);
+    setPanelType("apps");
+    setEntity(currentApp.name);
+  };
+
+  const closePanel = () => {
+    history.push(`/models/${modelName}`);
```

Comment:
If you watch the entries when closing the panel it loses it's mind creating new entries and rendering.

I think that you will only need to set the `panel` and `entity` to empty. But this will involve the discussion from the other qa issue below.